autorel/clients/github_client.py

```diff
@@ -15,46 +15,46 @@
 
 class Github:
 
-    '''
+    """
         Client interacting with the Github REST API
-    '''
+    """
 
     def __init__(self, endpoint, port, token_or_login, password, clientID,
                  clientSecret, userAgent, timeout):
-        self._authorizationHeaders = {}
+        self._headers = {}  # HTTP headers
```

Comment:
From Gitter:

----------------------------

MrAnno:
I'm perfectly OK with setting headers, but why don't you declare `headers` as a local variable? Do you want to use it outside of the constructor?

----------------------------

black-perl:
No I don't :+1: